lib/std/debug.zig

```diff
@@ -222,7 +222,8 @@ pub fn unlockStderrWriter() void {
 /// Print to stderr, unbuffered, and silently returning on failure. Intended
```

Comment:
I think this needs to be updated. Here's one option:

```suggestion
/// Print to stderr, silently returning on failure. Intended
```

Reply:
it's still unbuffered. the unlock function flushes.

Reply:
Ah, sorry. Thanks for clarifying